src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -90,9 +91,11 @@
 import org.apache.cassandra.utils.Throwables;
 
 import static java.lang.Math.max;
+import static org.apache.cassandra.cql3.statements.RequestValidations.invalidRequest;
 
 public class QueryController implements Plan.Executor, Plan.CostEstimator
 {
+    private static final String INDEX_MAY_HAVE_BEEN_DROPPED = "An index may have been dropped.";
```

Comment:
We might want to make this public and use it in the test, instead of copying the text message, as we often do with other error messages.